ckanext/switzerland/plugins.py

```diff
@@ -223,16 +223,7 @@ def before_map(self, map):
         return map
 
 
-class OgdchMixin(object):
```

Comment:
The Mixin was just used to read in the format mapping, since this is done differently now, it is no longer needed. The other changes in this file all relate to the fact that the format mapping does not need to be passed as an argument any more.